### PR TITLE
fix: Only increment whitelist counter if adding a new denom

### PR DESCRIPTION
**Pull request by apollo-sturdy**
Fixes Oak audit issue 5

### CODE REVIEW

**Review by piobab — APPROVED** [review]
LGTM

File: contracts/incentives/tests/test_whitelist.rs [hunk 1]
```diff
@@ -350,3 +358,48 @@ fn cannot_whitelist_more_than_max_limit() {
...
+
+    // Query whitelist
+    let whitelist: Vec<(String, Uint128)> = th_query(deps.as_ref(), QueryMsg::Whitelist {});
+    assert_eq!(whitelist, vec![("umars".to_string(), Uint128::new(3))]);
+
+    // Check whitelist count
+    let whitelist_count = WHITELIST_COUNT.load(&deps.storage).unwrap();
+    assert_eq!(whitelist_count, 1);
+
+    // Remove one denom from whitelist, and add a new one, should work
```
Comment (pacmanifold) [inline contracts/incentives/tests/test_whitelist.rs:384]: erroneous comment

**Review by pacmanifold — APPROVED** [review]
just one comment that is wrong

File: contracts/incentives/tests/test_whitelist.rs (hunk 1 above)
Comment (pacmanifold) [inline contracts/incentives/tests/test_whitelist.rs:384]:
```suggestion
    // Remove non-whitelisted denom, should error
```

**Pull request merged**